DayTomato/Xamarin.Android.Maps.Utils/Additions/AboutAdditions.txt

```diff
@@ -0,0 +1,48 @@
+﻿Additions allow you to add arbitrary C# to the generated classes
```

Comment:
Is it possible to omit this whole folder from the commit? It looks like we're including the whole project source or Xamarin.Android.Maps.Utils with this.